### PR TITLE
Feat/edit job list job

### PR DESCRIPTION
**Pull request by chiralcentre**

### CODE REVIEW

File: src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public class AddApplicationCommand extends Command {
     public static final String MESSAGE_EXCEEDS_VACANCY = "The number of accepted candidates already meets the"
             + " stipulated vacancy.\n To accept more candidates, the vacancy for the job can be increased via the"
             + " edit_job command, or change status for existing application(s) to this job via status command to"
-            + " a status other than ACCEPTED. You can retrieve vacancy left via slots_left command";
+            + " a status other than ACCEPTED.\n You can retrieve vacancy left via slots_left command";
```
Comment (logical-1985516) [inline src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java:42]: Should there be a whitespace after "\n" and should it be "vacancies"?

File: src/main/java/seedu/hirehub/model/ModelManager.java [hunk 2]
```diff
@@ -284,16 +283,21 @@ public int countVacancy(String jobTitle) {
...
         for (Application app : applicationList) {
             if (app.getJob().isSameJob(jobToFind) && app.getStatus().equals(new Status("ACCEPTED"))) {
                 countAccepted += 1;
             }
         }
+        return countAccepted;
+    }
+
+    @Override
+    public int countRemainingVacancy(String jobTitle) {
```
Comment (logical-1985516) [inline src/main/java/seedu/hirehub/model/ModelManager.java:297]: Should this also be (Job jobToFind)?

**Review by logical-1985516 — APPROVED** [review]
Other than a few nits, LGTM

File: src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java (hunk 1 above)
Comment (acekhoon) [reply src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java:42]: this one fixed alr:)

File: src/main/java/seedu/hirehub/logic/commands/ListApplicationCommand.java [hunk 3]
```diff
@@ -6,7 +6,7 @@
 import seedu.hirehub.model.Model;
 
 /**
- * Lists all persons in the address book to the user.
+ * Lists all jobs in the address book to the user.
```
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/main/java/seedu/hirehub/logic/commands/ListApplicationCommand.java]: fixed

File: src/main/java/seedu/hirehub/model/ModelManager.java (hunk 2 above)
Comment (chiralcentre) [reply src/main/java/seedu/hirehub/model/ModelManager.java:297]: hmm doesn't really matter in this case

**Pull request merged**